src/python/pcmdi/meanabs_xy.py

```diff
@@ -10,5 +10,8 @@ def compute(dm, do):
             "Absolute Difference Between Model And Observation",
             "Contact": "Peter Gleckler <gleckler1@llnl.gov>",
         }
-    mae = MV.average(MV.absolute(MV.subtract(dm, do)))
+    absdif = MV.absolute(MV.subtract(dm, do))
+    mae = cdutil.averager(absdif,axis='xy',weights='weighted')
```

Comment:
Are you positive we only send `xy` data to this? never any `tz` ? `weights="weighted"` is the default so probably not needed.